dtc/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Andy Gross <andy.gross@linaro.org>
+
+pkgname=dtc
+pkgver=1.4.4
```

Comment:
That makes it hard to test locally.  This behavior changed from a few weeks ago (the last time I compiled dtc).  I don't see warnings during build.  Seems odd to me.